tests/test_datacard.py

```diff
@@ -0,0 +1,38 @@
+import audbcards
+import pytest
+import difflib
+import numpy as np
+
+from audbcards.core.dataset import create_datasets_page
+
+
+def test_datacard_from_template_lines_similar(db, default_template):
```

Comment:
We are always using templates here, so I would change the name to

```suggestion
def test_datacard_lines_similar(db, default_template):
```